cypress/support/local-setup.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+occ group:add shareTestGroup
+occ group:adduser shareTestGroup alice
+occ group:adduser shareTestGroup bob
```

Comment:
Would be nice to have this as a cypress command:

I should be fairly easy to add an ocs api call similar to https://github.com/nextcloud/text/blob/main/cypress/support/commands.js#L132-L145 calling this API:

https://docs.nextcloud.com/server/latest/developer_manual/_static/openapi.html#/operations/provisioning_api-users-add-to-group